lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -84,7 +84,7 @@ def acts_as_list_top
             end
 
             def acts_as_list_class
-              self.class
+              ::#{self.name}
```

Comment:
Sure! There's a lot going on in this little change, can you be a little more specific with your question? Are you asking about the change from `self.class` to `self.name`, or are you asking about the syntax madness of `::#{}`?